deployment/deployment.json

```diff
@@ -1883,7 +1883,7 @@
         "status": "prod",
         "versions": {
           "schema": "1.3.0",
-          "subgraph": "1.1.2",
+          "subgraph": "1.1.3",
```

Comment:
could you also update the protocol getter to something along the lines of this so it will update the version when grafting? https://github.com/messari/subgraphs/pull/1167/files#diff-0c9712728b0d25a41e2e900556de984d5df37b3b5220362a0f6e97c899a0b13eR318

Reply:
I haven't done this before, do you have a sample subgraph that is doing this?

Reply:
this si what i am talking about: https://github.com/messari/subgraphs/pull/1167/files#diff-0c9712728b0d25a41e2e900556de984d5df37b3b5220362a0f6e97c899a0b13eR318